src/SFML/Graphics/RenderTextureImplFBO.cpp

```diff
@@ -160,11 +160,11 @@ RenderTextureImplFBO::~RenderTextureImplFBO()
     }
 
     // Move all frame buffer objects to stale set
-    for (std::map<Uint64, unsigned int>::iterator iter = m_frameBuffers.begin(); iter != m_frameBuffers.end(); ++iter)
-        staleFrameBuffers.emplace(iter->first, iter->second);
+    for (auto& [k, v] : m_frameBuffers)
+        staleFrameBuffers.emplace(k, v);
 
-    for (std::map<Uint64, unsigned int>::iterator iter = m_multisampleFrameBuffers.begin(); iter != m_multisampleFrameBuffers.end(); ++iter)
-        staleFrameBuffers.emplace(iter->first, iter->second);
+    for (auto& [k, v] : m_multisampleFrameBuffers)
+        staleFrameBuffers.emplace(k, v);
```

Comment:
Maybe use more expressive variable names, such as `contextId` and `frameBufferId` or so?
That's how they are named [here](https://github.com/SFML/SFML/blob/master/src/SFML/Graphics/RenderTextureImplFBO.cpp#L91-L93), for example.

Reply:
Good point, will change.